ui/app/styles/core/tags.scss

```diff
@@ -1,26 +1,14 @@
 .tag:not(body) {
-  background-color: lighten($grey-light, 17%);
+  background-color: $ui-gray-100;
   border-radius: 2px;
-  color: $grey-dark;
+  color: $grey;
   height: auto;
   padding: 0 $size-10;
   margin-right: 0.5rem;
   font-weight: normal;
 
   code {
-    color: $grey-dark;
-  }
-}
-
-.tag.is-outlined {
```

Comment:
So I think we should check the secrets list page where is shows the type of backend in the header - I think that's where this was used...

Reply:
Yep, this removes that. It was the only place we had it and it looks good with the normal tags too
![image](https://user-images.githubusercontent.com/111194/42469885-379c4364-8376-11e8-8c0b-a556bc356c5c.png)